contract-map.json

```diff
@@ -233,5 +233,12 @@
   "0x6090A6e47849629b7245Dfa1Ca21D94cd15878Ef": {
     "name": "ENS Registrar",
     "logo": "ens.svg"
+  },
+  "0x617b3f8050a0BD94b6b1da02B4384eE5B4DF13F4": {
```

Comment:
This address is not the same as [the one identified by Etherscan](https://etherscan.io/token/0x1b5f21ee98eed48d292e8e2d3ed82b40a9728a22). Can you provide a citation that this is the official address?